src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -285,7 +286,19 @@ export default class DefaultAudioVideoController implements AudioVideoController
               new SetLocalDescriptionTask(this.meetingSessionContext),
               new FinishGatheringICECandidatesTask(this.meetingSessionContext),
               new SubscribeAndReceiveSubscribeAckTask(this.meetingSessionContext),
-              new SetRemoteDescriptionTask(this.meetingSessionContext),
+              ...(this.meetingSessionContext.meetingSessionConfiguration.attendeePresenceTimeoutMs >
```

Comment:
Is splat/array necessary here? Each array in the ternary expression has one thing in it...

Reply:
Right, `...[]` is unnecessary. I udpated PR